### PR TITLE
Add idle worker count

### LINKED ISSUE #109: Add idle worker count

### PR DESCRIPTION
**Pull request by jpabadir**
closes #109

### CODE REVIEW

**Comment by JadenBalogh** [comment]
I've made a bunch of changes, so this should be ready for review! It's all specific to workers but I've left some comments on where things might need to be refactored later to support idle markers for any unit type.

File: Assets/Scripts/Game/Entity/MobileUnit.cs [hunk 1]
```diff
@@ -41,18 +44,6 @@ protected virtual void Update()
...
-        Vector2 direction = (position - transform.position).normalized;
-        Move(direction, moveSpeed);
-    }
-
-    protected virtual void Move(Vector2 direction, float speed)
-    {
-        rigidbody2D.velocity = speed * direction;
-        isMoving = true;
-    }
-
```
Comment (jpabadir) [inline Assets/Scripts/Game/Entity/MobileUnit.cs:55]: Not a big deal but these two methods only moved to L75, so they show as a big change in the PR. Is there a reason you moved them?

Comment (jpabadir) [reply Assets/Scripts/Game/Entity/MobileUnit.cs:55]: Maybe I moved them btw, I can't remember. I just mean "is there a reason they moved?"

Comment (JadenBalogh) [reply Assets/Scripts/Game/Entity/MobileUnit.cs:55]: Just maintaining the method order we use throughout the codebase which is: built-in Unity methods first, then custom methods after. In this case they were before `FixedUpdate()`.

**Pull request merged**